tests/test-cases/Group1-Docker-Commands/1-11-Docker-RM.robot

```diff
@@ -47,6 +47,12 @@ Remove a running container
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} rm ${container}
     Should Be Equal As Integers  ${rc}  1
     Should Contain  ${output}  Error response from daemon: You cannot remove a running container. Stop the container before attempting removal or use -f
+
+Remove a container created from unknown executable
```

Comment:
Let's move this test to the botton and update its title so we're consistent with `1-11-Docker-RM.md`.